### ISSUE TITLE
GUI problem: does not correctly show a grid of devices

### ISSUE THREAD

**Issue by metaphori** [AlchemistSimulator/Alchemist#73]
**On:** commit ab276ee (Alchemist 8.0.0)
**Verified absence on:** Alchemist 7.0.1 (plus commits from #72 )

When I start Alchemist with the following simulation configuration, I only see one node (or possibly all nodes located in a single point), instead of a grid of nodes.

```yaml
network-model:
  type: EuclideanDistance
  parameters: [2.0]

incarnation: scafi

positions:
  type: Continuous2DEuclidean

displacements:
- in:
    type: Grid
    parameters: [0, 0, 20, 20, 1, 1, 0.8, 0.8]
```

**Comment by metaphori** [comment]
The problem seems to be located in:  alchemist/alchemist-loading/src/main/kotlin/it/unibo/alchemist/loader/displacements/Grid.kt

**Comment by DanySK** [comment]
@metaphori, I cannot reproduce it on 02276e423a5cfa35ffb17de07bd2aa44928f2859. I added a test, and it passes, creating 400 nodes with distinct positions. Where do you find the problem in the file you suggested? I don't see what's wrong.

**Comment by DanySK** [comment]
By the way you should use `ConnectWithinDistance` in place of `EuclideanDistance`, which is deprecated. Moreover, the positions entry is ignored entirely (position generation is now directly delegated to the environment).

**Comment by metaphori** [comment]
@DanySK You're right. Indeed, I checked the number and position of nodes, and they seem correct. It must be a GUI problem: I see just one node, and by pressing 'L' for links, I see none.

**Comment by DanySK** [comment]
I'm able to reproduce it.

**Comment by DanySK** [comment]
Should be solved by 80ec787. I'm not closing as apparently there is also an issue with zooming and panning.

**Comment by DanySK** [comment]
This issue is now solved. Thanks for reporting.
